src/main/java/it/gov/pagopa/wispconverter/technicalsupport/service/mapper/ReEventMapper.java

```diff
@@ -2,17 +2,25 @@
 
 import it.gov.pagopa.wispconverter.technicalsupport.repository.model.ReEventEntity;
 import it.gov.pagopa.wispconverter.technicalsupport.service.model.ReEventDto;
+import it.gov.pagopa.wispconverter.technicalsupport.util.Constants;
 import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
 import org.mapstruct.ReportingPolicy;
 
 import java.util.List;
 
 @Mapper(componentModel = "spring", unmappedTargetPolicy = ReportingPolicy.IGNORE)
-public interface ReEventMapper {
+public abstract class ReEventMapper {
 
+    @Mapping(target = "httpHeaders", expression = "java(obfuscateSecrets(reEventEntity.getHttpHeaders()))")
+    public abstract ReEventDto toReEventDto(ReEventEntity reEventEntity);
 
-    ReEventDto toReEventDto(ReEventEntity reEventEntity);
-    List<ReEventDto> toReEventDtoList(List<ReEventEntity> reEventEntity);
-
+    public abstract List<ReEventDto> toReEventDtoList(List<ReEventEntity> reEventEntity);
 
+    public String obfuscateSecrets(String httpHeaders) {
```

Comment:
In certain situations, like the `paaInviaRT` send process, if the communication fails because of an error on Forwarder or on creditor institution, it may be useful to know whether the wisp-converter sends this header or not.